tsconfig.base.json

```diff
@@ -6,12 +6,12 @@
     "downlevelIteration": true,
     "esModuleInterop": true,
     "jsx": "react",
-    "lib": ["DOM", "ES2015", "ES6", "DOM.Iterable", "ScriptHost"],
+    "lib": ["DOM", "ES2020", "ES6", "DOM.Iterable", "ScriptHost"],
     "moduleResolution": "Node",
     "preserveConstEnums": true,
     "sourceMap": true,
     "strict": true,
-    "target": "ES5",
+    "target": "ES6",
```

Comment:
These changes are maybe why we have differences in linted import requirements (e.g. using `new mediaQuery` etc)?

Reply:
Yea thats likely, it took me a while to follow the traceback of all the build / `npm run start` errors to get to this point where I knew I had to change the target. I looked at the "can i use" or w.e. that website is that lets me check min browser support versions and ES6 seems fine to me.